src/TipBox.vue

```diff
@@ -31,6 +33,10 @@
         type: String,
         default: null
       },
+      iconSize: {
+        type: String,
+        default: null
+      },
```

Comment:
If no semantic nor alphabetical order, then default to adding at the end.

In this case, we can put `iconSize` after `icon` ⁠— grouping `icon*` props and then sorting by alphabetical order.